ts/features/wallet/onboarding/common/searchBank/SearchBankComponent.tsx

```diff
@@ -70,22 +68,22 @@ export const SearchBankComponent: React.FunctionComponent<Props> = (
 
   return (
     <>
-      {!props.isLoading && (
-        <Label color={"bluegrey"}>{I18n.t("wallet.searchAbi.bankName")}</Label>
-      )}
-      <Item>
-        <Input
-          value={searchText}
-          autoFocus={true}
-          onChangeText={handleFilter}
-          disabled={props.isLoading}
-          placeholderTextColor={IOColors.bluegreyLight}
-          placeholder={
-            props.isLoading ? I18n.t("wallet.searchAbi.loading") : undefined
-          }
-        />
-        {!props.isLoading && <IconFont name="io-search" />}
-      </Item>
+      <LabelledItem
+        type="text"
+        label={
+          !props.isLoading ? I18n.t("wallet.searchAbi.bankName") : undefined
+        }
+        inputProps={{
+          value: searchText,
+          autoFocus: true,
+          onChangeText: handleFilter,
+          disabled: props.isLoading,
+          placeholder: props.isLoading
+            ? I18n.t("wallet.searchAbi.loading")
+            : undefined
+        }}
+        icon={!props.isLoading ? "io-search" : undefined}
```

Comment:
`props.isLoading` is looked up three times here, I'd consider destructuring to make it more concise `const { isLoading } = props` and possibly only non-negative conditions in ternaries to improve readability.